### PR TITLE
Trigger fetch preview on initialize, not getContent

### PR DESCRIPTION
**Pull request by mattheu**
#301

Removed getContent method - no need provide our own version of this method. Also removed tests for getContent.

Backported fix to 4.1 compatability shim

### CODE REVIEW

File: inc/class-shortcode-ui.php [hunk 1]
```diff
@@ -64,7 +64,11 @@ public function get_shortcode( $shortcode_tag ) {
 
 	}
 
-	public function action_after_setup_theme() {
+	/**
+	 * Action admin scripts.
+	 */
+	public function action_admin_enqueue_scripts() {
```
Comment (danielbachhuber) [inline inc/class-shortcode-ui.php:70]: Why did you change this action?

Comment (mattheu) [inline inc/class-shortcode-ui.php:70]: This wasn't working at all. `after_setup_theme` must be too early.

**Comment by mattheu** [comment]
editor styles CSS file wasn't loading at all. So I removed a bunch of they styles since we don't seem to need them!

Kept styles for error messages.

When shortcode preview is empty - just display the formatted shortcode.

I wonder if we should do this for errors too? In that case we can simply return a JSON error if the shortcode content is empty.

**Comment by danielbachhuber** [comment]
> When shortcode preview is empty - just display the formatted shortcode.

Screen grab?

> I wonder if we should do this for errors too? In that case we can simply return a JSON error if the shortcode content is empty.

Could. I don't have a strong preference. Your call.

**Comment by mattheu** [comment]
image

I think we should keep the error message as is. If you're seeing this - theres a problem ;)
